core/src/test/java/com/radixdlt/api/engine_state/EntityInfoTest.java

```diff
@@ -224,7 +224,12 @@ public void engine_state_api_returns_uninstantiated_object_info() throws Excepti
       assertThat(collectionsNames)
           .isEqualTo(List.of("resource_vault", "resource_preference", "authorized_depositor"));
 
-      assertThat(objectInfo.getAttachedModules()).isEmpty();
+      final var attachedModuleIds =
+          Lists.transform(
+              objectInfo.getAttachedModules(),
+              ObjectEntityInfoAllOfAttachedModules::getAttachedModuleId);
+      assertThat(attachedModuleIds)
+          .isEqualTo(List.of(AttachedModuleId.ROLEASSIGNMENT, AttachedModuleId.METADATA));
```

Comment:
_(note from author)_

This is a legit change of behavior - previously-hardcoded return did not contain the right attached modules.